From d5643cd0c49056e10a995dd9a1b943dab6f792aa Mon Sep 17 00:00:00 2001
From: Dave Eddy <dave@daveeddy.com>
Date: Wed, 22 Aug 2018 11:38:26 -0400
Subject: [PATCH] joyent/node-manta#347 mmkdir test fails when MANTA_TEST_ROLE
 is unset Reviewed by: Robert Mustacchi <rm@joyent.com> Approved by: Kelly
 McLaughlin <kelly.mclaughlin@joyent.com>

---
 test/mmkdir.test.js | 21 ++++++++++-----------
 1 file changed, 10 insertions(+), 11 deletions(-)

diff --git a/test/mmkdir.test.js b/test/mmkdir.test.js
index e2063cf..7651918 100644
--- a/test/mmkdir.test.js
+++ b/test/mmkdir.test.js
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Joyent, Inc.
+ * Copyright 2018 Joyent, Inc.
  */
 
 /*
@@ -85,16 +85,15 @@ if (process.env.MANTA_TEST_ROLE) {
                   });
            });
     });
-}
-
 
-test('cleanup: rm test directory ' + TESTDIR, function (t) {
-    // Sanity checks that we don't `mrm -r` a non-test dir.
-    assert.ok(TESTDIR);
-    assert.ok(TESTDIR.indexOf('node-manta-test') !== -1);
+    test('cleanup: rm test directory ' + TESTDIR, function (t) {
+        // Sanity checks that we don't `mrm -r` a non-test dir.
+        assert.ok(TESTDIR);
+        assert.ok(TESTDIR.indexOf('node-manta-test') !== -1);
 
-    forkExecWait({ argv: [ MRM, '-r', TESTDIR ]}, function (err) {
-        t.ifError(err, err);
-        t.done();
+        forkExecWait({ argv: [ MRM, '-r', TESTDIR ]}, function (err) {
+            t.ifError(err, err);
+            t.done();
+        });
     });
-});
+}
-- 
2.21.0

